### DISCUSSION TITLE
Update cards to display a random sentence containing the "study morph" from the bank of cards with all known morphs

### DISCUSSION THREAD

**Discussion by xofm31** [mortii/anki-morphs#233] [Ideas]
This idea is probably out of scope for this add-on. But it has so many of the Ankimorphs elements, I thought I would write it up here anyway. Feel free to say that it is out of scope and archive/delete it.

Right now, with only a few exceptions, I have one card for each of the morphs (I selected "move new cards without unknown words to the end of the due cue"). Seeing words in multiple contexts is known to be very good for learning. Especially for harder words, I would really like to be able to see more than just one context. For some of my cards, I find that I know what the English is based on the shape of the sentence, rather than actually reading and understanding the word. 

At this point, I have thousands of cards with all known words sitting at the end of the due cue. Some of these cards have additional contexts for the words that I have trouble with. Even for easier words, being able to see new contexts would be helpful (reading and listening more of course is also helpful, but using Anki helps me to focus on the meanings of the individual study words as opposed to the whole sentence/paragraph). I am not learning a language with inflections, but being able to see different sentences with different inflections for a given lemma seems like it would be very helpful as opposed to seeing the same single form of the word every time.

So here's my suggestion: be able to select a bunch of cards in the browser, click on "Find new sentences", and those cards would be re-populated with other sentences containing the `study word` from the bank of cards with all known morphs. This would allow me to benefit from the full bank of cards that I've collected. I would want it to choose the sentence randomly, with it somewhat favoring sentences that have a lower score with the new `card due` algorithm. 

As you can see, this idea uses a lot of Ankimorph components, including the Ankimorph parses, the calculation of how many unknown morphs there are on a card, and the user preference for the kinds of cards that they like to see. Of course, there are many complexities that would have to be addressed. There is a lot more than just the sentence that would need to be copied into the card, and some of it Ankimorphs isn't aware of. Where would the content from the current card go?

Maybe if you like the concept, you will have a better way to implement it that didn't involve copying content between cards.

**Comment by mortii** [comment]
Hmm, this use case might be a little too narrow. If you want to see more the morph in more contexts, could you not just search for that particular morph in the browser and go through the cards it found/choose one randomly?

**Reply by xofm31** [reply]
Yes, that is what I do today. It would be easier to do it automatically for a selection of cards, but I can continue to do it manually.

**Reply by mortii** [reply]
We can reopen this later if someone else also requests this feature :+1:
